### ISSUE TITLE
pH7Software/pH7-Internationalization-Italian-Language

### ISSUE THREAD

**Issue by DEATHJ0KER** [pH7Software/pH7-Internationalization#20]
Hi, I have completely translated the file. Po and I follow step-by-step script to add the Italian language to your plug-ins. I created the It_IT folder and put the required files in the default location. By clicking on the Italian flag that comes to be generated recognizing the language added, the server returns me Error 500. The Global. po file in English works well when used with other languages. What could be the problem?

**Comment by pH-7** [comment]
Awesome @DEATHJ0KER! I will try to investigate your error and I will add your translation to the repo as well. I really appreciate it!! Thanks a lot

**Comment by pH-7** [comment]
Concerning your 500 error, can you please enable the the "development" mode by going to -> Admin Panel -> Tools -> Environment Mode), and then try again to switch to Italian language and give me the error you see there.

**Comment by DEATHJ0KER** [comment]
Hello, Thank you for taking this translation into consideration.
I solved using a google translator button, translates into all the languages of the world LOL³
Joke!

Of course, I followed your instructions and to avoid further errors I set the direct dns on cloudflare on development (because it causes error 500 or 502 if it does not use the original dns of my namecheap hosting) as well as the development mode in PH7CMS script was executed. 
This error is returned:

**Fatal error: Namespace declaration statement has to be the very first statement or after any declare call in the script in /home/vxdmdfxx/public_html/dating/_protected/app/langs/it_IT/language.php on line 9**

I state that I have no idea how to correct this error :(

**Comment by DEATHJ0KER** [comment]
Some advice if possible…

- Geolocation does not propose to meet people in the vicinity of my city, but in the capital of my country which is 1000 km away. 
Is it possible to get a better result using third-party api?

- I have a package of 20,000 user profiles.The automatic installation is not successful.I have divided the file into many small packages and the installation is not completed. I tried to load them in other dating scripts, the same problem with dedicated software. Is there no solution to my problem or can I try another solution? 
Thank you

**Comment by pH-7** [comment]
> Geolocation does not propose to meet people in the vicinity of my city, but in the capital of my country which is 1000 km away. 
> Is it possible to get a better result using third-party api?

@DEATHJ0KER I answered to you here on a new thread https://github.com/pH7Software/pH7-Social-Dating-CMS/issues/421#issuecomment-456672528 (because it wasn't related to your initial question).

Good luck!

**Comment by DEATHJ0KER** [comment]
The translation into the Italian language was performed scrupulously & complete.
Thanks for the support and for your availability, I wish you the greatest success for your work, because pH7 CMS deserve his place in the ranking of the best Social Dating $cript$!

**Comment by pH-7** [comment]
Thank you @DEATHJ0KER! I appreciate your kind message!
Good luck, you too!

**Comment by DEATHJ0KER** [comment]
[it_IT.zip](https://github.com/pH7Software/pH7-Internationalization/files/2857548/it_IT.zip)
Hello, sorry for my delay, I was very busy. I finally solved the problems related to the translation of the Italian language. One mistake was due to the syntax of TimeZone, Europe/Roma corrected with Europe/Rome. The other problem was due to the encoding of the config.ini file. He used UTF8-BOM hehehe :| Excuse my lack of attention... In any case, the error has finally been solved and is 100% working. If the translation of the script installation is also necessary, just let me know, I would be very happy to provide the installation in the Italian Language :)
